safe-global/web-core
# fix: improve `EnhancedTable` + hide send on mobile

## What it solves

Resolves #888

## How this PR fixes it

- The "Send" button in the assets table is hidden when `!isGranted || isMobile`
- If actions are hidden, i.e. because the user is not granted, the header is now hidden as well. `EnhancedTable` now hides them via CSS instead of JS.

## How to test it

Open the Safe on mobile and observe that the assets "Send" button is never visible, including when connected. All other tables according to the following conditions:

- Address book: always, (no "Send" on mobile)
- Owners: edit always, others when granted
- Spending limit: when granted

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/195336212-7b8e1633-66d0-4e42-8c6b-fe6b18dd0ebd.png)

![image](https://user-images.githubusercontent.com/20442784/195346490-2f7f19b9-72cc-47c2-8e45-c80b37dc936f.png)

![image](https://user-images.githubusercontent.com/20442784/195336364-8997fed2-ee9c-4241-8d64-7f99857d59db.png)

![image](https://user-images.githubusercontent.com/20442784/195336416-fb78184c-f437-4cfe-be12-488ba693e3b1.png)
